### ISSUE TITLE
Installation failure in virtualenv on CentOS 7.2

### ISSUE THREAD

**Issue by verdurin** [testing-cabal/funcsigs#21]
I'm trying to install Elasticluster in a virtualenv on CentOS 7.2.

`funcsigs` is one of the dependencies. However, when I run `python setup.py install`, there is an error with `funcsigs`:

```
Processing dependencies for elasticluster==1.2.1.rc1
Searching for funcsigs>=0.4
Reading https://pypi.python.org/simple/funcsigs/
Best match: funcsigs 1.0.1
Downloading https://pypi.python.org/packages/ab/5b/a6dff630fe5b68a4d2a049b6d95b51ad1510fb72e9606d656feb2c34efd8/funcsigs-1.0.1.tar.gz#md5=c2d84de116ce1aba0dc844a359cd20dd
Processing funcsigs-1.0.1.tar.gz
Writing /tmp/easy_install-hpmqdV/funcsigs-1.0.1/setup.cfg
Running funcsigs-1.0.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-hpmqdV/funcsigs-1.0.1/egg-dist-tmp-60yD2x
error: Setup script exited with error in funcsigs setup command: Invalid environment marker: python_version<"2.7"
```

**Comment by verdurin** [comment]
At first I thought it might be down to `pip` being too old, so I upgraded that, but this made no difference:

```
pip list
ansible (1.9.6)
Babel (2.3.4)
boto (2.39.0)
coloredlogs (5.0)
configobj (5.0.6)
debtcollector (1.3.0)
ecdsa (0.13)
elasticluster (1.2.1rc1)
google-api-python-client (1.5.0)
httplib2 (0.9.2)
humanfriendly (1.44.7)
iso8601 (0.1.11)
Jinja2 (2.8)
keystoneauth1 (2.6.0)
MarkupSafe (0.23)
monotonic (1.0)
netaddr (0.7.18)
netifaces (0.10.4)
oauth2client (2.0.2)
oslo.i18n (3.6.0)
oslo.serialization (2.4.0)
oslo.utils (3.8.0)
paramiko (1.16.0)
pbr (1.9.1)
pip (8.1.1)
prettytable (0.7.2)
pyCLI (2.0.3)
pycrypto (2.6.1)
python-gflags (3.0.4)
python-novaclient (4.0.0)
pytz (2016.4)
PyYAML (3.11)
requests (2.9.1)
setuptools (0.9.8)
simplejson (3.8.2)
six (1.10.0)
uritemplate (0.6)
voluptuous (0.8.11)
```

**Comment by verdurin** [comment]
Hmm, upgrading `setuptools` via `pip` appeared to fix this. Closing for now.

**Comment by rbtcollins** [comment]
What version of setup tools produces this error?

**Comment by verdurin** [comment]
`0.9.8`

**Comment by rbtcollins** [comment]
Thanks. I'll add a specifier as a setup requires to signal that a newer
version is required.

**Comment by rbtcollins** [comment]
Released 1.0.2
